src/modules/menu/components/menu/MenuTray.vue

```diff
@@ -77,10 +84,45 @@ export default {
     computed: {
         ...mapState({
             activeLayers: (state) => state.layers.activeLayers,
+            activeKmlLayer: (state) => state.drawing.activeKmlLayer,
+            hostname: (state) => state.ui.hostname,
         }),
         showLayerList() {
             return this.activeLayers.length > 0
         },
+        disableDrawing() {
+            if (
+                DISABLE_DRAWING_MENU_FOR_LEGACY_ON_HOSTNAMES.some(
+                    (hostname) => hostname === this.hostname
+                )
+            ) {
+                if (this.activeKmlLayer && this.activeKmlLayer.isLegacy()) {
+                    return true
+                }
+            }
+            return false
+        },
+    },
+    watch: {
+        disableDrawing(disableDrawing) {
+            if (disableDrawing) {
+                this.disableDrawingTooltip = tippy('#drawSectionTooltip', {
+                    content:
+                        'You are not allowed to edit drawing created by map.geo.admin.ch on the test.map.geo.admin.ch.\n' +
```

Comment:
Just a minor picky thing, sorry:
Probalby we could say "It is not possible to..." instead of "you are not allowed".
As "you are not allowed" implies, that it is possible given the correct permissions to do so.

Reply:
👍🏼

Reply:
Done with translation